Content.Server/Chat/Commands/MOTDCommand.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Server.Administration;
+using Content.Server.Administration.Logs;
+using Content.Server.Administration.Managers;
+using Content.Server.Chat.Managers;
+using Content.Server.Chat.Systems;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Content.Shared.Chat;
+using Content.Shared.Database;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Console;
+
+namespace Content.Server.Chat.Commands;
+
+[AnyCommand]
```

Comment:
This should be using a proper command flag.

Reply:
What do you mean by this? The commands I'm going off of just use AnyCommand.

Reply:
sloth may be talking about the [AdminCommand] flag

Reply:
That doesn't let normal players use the get function of the command. Is there a way to alias two different commands to the same name depending on who's running the command?

Reply:
I suppose I can make three commands, motd, get-motd, and set-motd and just relay between the first and the latter.

Reply:
Done-ish.  `AdminCommand` is now used for the set command. The alias and getter commands still need to be AnyCommand so normal players can use them.